comet/utils/redis.py

```
@@ -0,0 +1,191 @@
+            logger.error(f"Redis DELETE error for key {key}: {e}")
+            return False
+
+    async def exists(self, key: str) -> bool:
+        if not self.is_connected():
+            return False
+        
+        try:
+            result = await self.redis.exists(key)
+            return bool(result)
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)